### PR TITLE
Updating tools/varvamp from version 1.2.0 to 1.2.1

### PR DESCRIPTION
**Pull request by gxydevbot**
Hello! This is an automated update of the following tool: **tools/varvamp**. I created this PR because I think the tool's main dependency is out of date, i.e. there is a newer version available through conda.

I have updated tools/varvamp from version 1.2.0 to 1.2.1.

**Project home page:** https://github.com/jonas-fuchs/varVAMP/releases

For any comments, queries or criticism about the bot, not related to the tool being updated in this PR, please create an issue [here](https://github.com/planemo-autoupdate/autoupdate/issues/new).

If you want to skip this change, close this PR without deleting the branch. It will be reopened if another change is detected.

Any commit from another author than 'planemo-autoupdate' will prevent more auto-updates.

To ignore manual changes and allow autoupdates, delete the branch.

### CODE REVIEW

**Comment by bgruening** [comment]
@jonas-fuchs, you changed the FASTA header line to include varVAMP- and dropped more specific information instead. Was this intended?

**Comment by jonas-fuchs** [comment]
Which info do you mean? I mainly changed the bed file format to be artic compatible and included that you can name your scheme. Indeed some information is lost in the main outputs, but e.g. the primer names and penalties you still have this info in the tsv and primer all files.

**Comment by bgruening** [comment]
Test are failing here for example because of that diff 

````
Output ambiguous_consensus:  different than expected, difference (using diff):
( /tmp/tmp2ak5l8smambiguous_consensus.fasta v. /tmp/tmpgu9586ieambiguous_consensus.fasta )
--- local_file
+++ history_data
@@ -1,2 +1,2 @@
->ambiguous_consensus
+>varVAMP_consensus
 tatcccgtrtycaractgayatccttattaayytgatgcaaccycgrcagcttgtkttccgrccygaagtyytstggaaycayccgatccagcgrgtyatacataatgagctggagcartactgccgwgcycgygctggycgytgyctkgargtkggsgcycayccaagatcyatyaatgayaacccyaatgtyytgcaccggtgcttyctycgcccggtyggdagagaygtmcagcgytggtaytcygccccsacycgyggyccagcggcyaaytgccgccgytcygcgctacgyggyytgccccctgtcgaycgyacmtaytgcttcgacgggttytcccgctgcgcytttgccgctgagacyggratygctttataytcactrcatgacctytggccytcggaygtygcggaggcya

**Comment by jonas-fuchs** [comment]
Renaming of the fasta header was intended. The name of the file hasn t changed giving the info which consensus it is. Sry about the trouble 😅

File: tools/varvamp/varvamp.xml [hunk 1]
```diff
@@ -53,7 +53,7 @@ results/
 #end if
 #if $mode.m_select == 'tiled' and $mode.scheme_outputs and 'primer_dimers' in $mode.scheme_outputs:
   ## ensure the unsolvable_primer_dimers.tsv file, which varVAMP creates only conditionally, exists in all cases, in which we try to discover it as an output
-  && cp -n dimers_fallback.tsv results/unsolvable_primer_dimers.tsv
+  && cp --update=none dimers_fallback.tsv results/unsolvable_primer_dimers.tsv
```
Comment (bgruening) [inline tools/varvamp/varvamp.xml:56]: this was a warning before

Comment (wm75) [reply tools/varvamp/varvamp.xml:56]: sorry, what was the warning? Cannot find it.

Comment (wm75) [reply tools/varvamp/varvamp.xml:56]: everything else looks easy enough to fix :-)

Comment (bgruening) [reply tools/varvamp/varvamp.xml:56]: That -n is deprecated and I should use the long version

**Comment by wm75** [comment]
handled in #6486

**Pull request closed without merging**